schema/vrs-source.yaml

```diff
@@ -240,6 +246,43 @@ definitions:
 
     required: [ "subject", "copies" ]
 
+  Genotype:
+    description: >-
```

Comment:
For just about all of our "Variation" data types we have a "Description" that shows right under the datatype main heading and a "Computational Definition". It seems that our Genotype is missing the former "general" description and this description is showing up as the "Computational Definition". For quality and consistency purposes we should follow the approach in "[Haplotype](https://vr-spec--394.org.readthedocs.build/en/394/terms_and_model.html#haplotype)" shown here

> Haplotypes are a specific combination of Alleles that are in-cis: occurring on the same physical molecule. Haplotypes are commonly described with respect to locations on a gene, a set of nearby genes, or other physically proximal genetic markers that tend to be transmitted together.
> 
> Computational Definition
> 
> A set of non-overlapping [Allele](https://vr-spec--394.org.readthedocs.build/en/394/terms_and_model.html#allele) members that co-occur on the same molecule.